### PR TITLE
Implemented code to display current OMS information on Settings page(…

### PR DESCRIPTION
**Pull request by disha1202**
…#1y2ract)

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -13,7 +13,10 @@
         <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
         <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
       </ion-item>
-
+      <!-- OMS information -->
+      <ion-item>
+        <ion-label>OMS : {{instanceUrl}}</ion-label>
```
Comment (dt2patel) [inline src/views/Settings.vue:18]: Put instance url in ion note and put it in the "end" slot. Remove colon

**Review by dt2patel — CHANGES_REQUESTED** [review]
Markup change

File: src/views/Settings.vue (hunk 1 above)
Comment (disha1202) [reply src/views/Settings.vue]: Done sir

**Review by dt2patel — APPROVED** [review]
Looks good

**Comment by adityasharma7** [comment]
Change on the login page too
https://github.com/hotwax/ionic-sdk/blob/main/src/views/Login.vue#L9

**Comment by disha1202** [comment]
> Change on the login page too https://github.com/hotwax/ionic-sdk/blob/main/src/views/Login.vue#L9

Updated the label on login page.

**Review by ymaheshwari1 — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**